integration-tests/load/ccip/ccip_test.go

```diff
@@ -131,33 +132,40 @@ func TestCCIPLoad_RPS(t *testing.T) {
 			finalSeqNrExecChannels)
 	}
 
+	evmSourceKeys := make(map[uint64]*bind.TransactOpts)
+	solanaSourceKeys := make(map[uint64]*solana.PrivateKey)
+	for ind := range *userOverrides.NumDestinationChains {
+		cs := env.AllChainSelectors()[ind]
+		other := env.AllChainSelectorsExcluding([]uint64{cs})
+		var mu sync.Mutex
+		for _, src := range other {
+			//todo: handle solana source keys
+			mu.Lock()
+			evmSourceKeys[src] = transmitKeys[src][ind]
+			mu.Unlock()
```

Comment:
The mutux has no effect from a single go routine:
```suggestion
		for _, src := range other {
			//todo: handle solana source keys
			evmSourceKeys[src] = transmitKeys[src][ind]
```